test/controllers/readings.test.js

```diff
@@ -0,0 +1,127 @@
+const chai = require('chai');
+const { saveDevice } = require('../../src/repos/devices');
+const { saveReading } = require('../../src/repos/readings');
+const { getReadings, getReadingsForDevice, addReading } = require('../../src/controllers/readings');
+const { isIncluded, buildDevice, buildReading } = require('../../src/utils');
+
+const { expect } = chai;
+
+describe('Controllers - readings', () => {
+  describe('addReading', () => {
+    let newDevice = {};
+    let newReading = {};
+    let context = {};
+
+    beforeEach(async () => {
+      newDevice = await saveDevice(buildDevice());
+      newReading = buildReading(newDevice._id);
+      context = {
+        request: {
+          body: newReading,
+        },
+        params: {
+          deviceId: newDevice._id,
+        },
+      };
+    });
+
+    it('should return the reading that was added', async () => {
+      await addReading(context);
+      expect(context.status).to.equal(201);
+      expect(context.body.success).to.equal(true);
```

Comment:
Use more specific expect/assert matcher: http://chaijs.com/api/assert/
For example: `assert.isTrue(context.body.success);` or `expect(context.body.success).to.be.true;` also destructure context for cleaner assertions `const { status, body: { success, result } } = context`